src/field_control_avatar.c

```diff
@@ -1060,3 +1091,114 @@ int SetCableClubWarp(void)
     SetupWarp(&gMapHeader, GetWarpEventAtMapPosition(&gMapHeader, &position), &position);
     return 0;
 }
+
+static bool32 TrySetUpWalkIntoSignpostScript(struct MapPosition *position, u32 metatileBehavior, u32 playerDirection)
+{
+    const u8 *script;
+
+    if (JOY_HELD(DPAD_LEFT | DPAD_RIGHT))
+        return FALSE;
+    if (playerDirection != DIR_NORTH)
+        return FALSE;
```

Comment:
These could also be combined into a single condition I think

Reply:
https://github.com/rh-hideout/pokeemerald-expansion/pull/5044/commits/b39258ebced39d8cf858c8f5551619fa5e5a271f